diff --git a/node_modules/react-native-web/src/vendor/react-native/VirtualizedList/index.js b/node_modules/react-native-web/src/vendor/react-native/VirtualizedList/index.js
index 97f0a95..9a4dbc8 100644
--- a/node_modules/react-native-web/src/vendor/react-native/VirtualizedList/index.js
+++ b/node_modules/react-native-web/src/vendor/react-native/VirtualizedList/index.js
@@ -747,7 +747,7 @@ class VirtualizedList extends StateSafePureComponent<Props, State> {
     if (this._scrollRef && this._scrollRef.getScrollableNode) {
       this._scrollRef.getScrollableNode().addEventListener('wheel',
           this.invertedWheelEventHandler,
-          { passive: true },
+          { passive: false },
       );
     } else {
       setTimeout(() => this.setupWebWheelHandler(), 50);
